skbio/util/_testing.py

```diff
@@ -8,6 +8,53 @@
 
 import os
 import inspect
+from nose import core
+from nose.tools import nottest
+from future.utils import PY3
+
+
+@nottest
+class TestRunner(object):
+    """Simple wrapper class around nosetests functionality.
+
+    Parameters
+    ----------
+    filename : __file__ attribute passed in from the caller. This tells the
+        tester where to start looking for tests.
+
+    Attributes
```

Comment:
argv no longer exists, but self._test_dir and self._filename are both used. They are both "private" though. Should I still remove the section?